### ISSUE TITLE
[unhandledRejection] Cannot set headers after they are sent to the client

### ISSUE THREAD

**Issue by mak33v** [nuxt/nuxt.js#14056]

**Comment by wa1id** [comment]
Fixed for me by upgrading from Nuxt 3.0.0 to Next 3.2.3 (internally upgraded nitropack from 1.0.0 to 2.2.3)

**Comment by shide1989** [comment]
@wa1id tried it but didn't work for me.

I fixed the issue for me by updating my vite config, as mentioned in https://github.com/nuxt/nuxt/issues/12748#issuecomment-1397234478:
```
export default defineNuxtConfig({
  vite: {
    server: {
      hmr: {
        protocol: 'ws'
      }
    }
  }
})
```
